string CMD
Player myPlayer 
----
uint8 Status
Player yourPlayer
Map map
